### PR TITLE
Update docs about how to run Salsah1 tests

### LINKED ISSUE #829: Documentation about how to run SALSAH1 tests
Where can I find the documentation about how to tun the browser tests for salsah1?

- Sipi needs to be started with `--config config/sipi.knora-test-config.lua`
- Knora webapi needs to be started with `-r`
- Salsah1 needs to be running and then the tests can be executed with `test` in sbt

### PR DESCRIPTION
**Pull request by tobiasschweizer**
closes #829

### CODE REVIEW

**Comment by tobiasschweizer** [comment]
@subotic Turns out that I have actually documented it.

There is probably some outdated stuff (e.g., `re-start` instead of `reStart`), could you please check?

**Comment by tobiasschweizer** [comment]
@subotic Could you add or modify the docker related docs? Thanks!

**Comment by subotic** [comment]
> `re-start` instead of `reStart`

actually both work. It is just a matter of personal preference.

I will check and update where needed.

**Comment by subotic** [comment]
Steps to add:

1. start and initialize graphdb-se
    - ./graphdb-se-docker-init-knora-test.sh
1. start the sipi container (with test config)
    - docker run -d --add-host webapihost:[local IP address] -v /tmp:/tmp -v $HOME:$HOME -p 1024:1024 dhlabbasel/sipi:develop /sipi/local/bin/sipi --config=/sipi/config/sipi.knora-test-docker-config.lua
1. start the webapi and salsah1 server
   - cd knora/webapi/ && sbt "run allowReloadOverHTTP" &
    - cd knora/salsah1/ && sbt run &
1. wait for the webapi server to start before we proceed with running the salsah tests
1. unzip chromedriver
    - cd knora/salsah1/lib/chromedriver/ && unzip chromedriver_linux64.zip
1. start built-in salsah1 browser tests
    - cd knora/salsah1/ && sbt headless:test

**Comment by subotic** [comment]
I've updated the documentation.

**Comment by tobiasschweizer** [comment]
@subotic SHould me reapply these changes to the new doc style?

**Comment by tobiasschweizer** [comment]
I will also look into https://github.com/dhlab-basel/Knora/issues/847 and see if I found out what the problem is.

**Comment by subotic** [comment]
> Should I reapply these changes to the new doc style?

What do you mean by this?

**Comment by tobiasschweizer** [comment]
the old RST docs were deleted, I would have to add the changes of this PR to the markdown files. 

Actually I have already fixed some of the Salsah tests. I will make a PR for that.

**Comment by subotic** [comment]
ah, I understand now. yes, please :-)

File: docs/rst/04-salsah/development/build-process.rst [hunk 1]
```diff
@@ -48,50 +51,57 @@ Then go back to the webapi root directory and use SBT to start the API server:
...
+    > reStart
 
-Then in another terminal, go to the SIPI project root directory and start the server:
+Then in another terminal, start the SIPI server by using Docker:
 
 ::
 
-    $ ./local/bin/sipi --config=config/sipi.knora-config.lua (for production)
-    $ ./local/bin/sipi --config=config/sipi.knora-test-config.lua (for running tests)
+    $ docker run -d --add-host webapihost:[local-IP-address] -v /tmp:/tmp -v $HOME:$HOME -p 1024:1024 dhlabbasel/sipi:develop /sipi/local/bin/sipi --config=/sipi/config/sipi.knora-docker-config.lua
```
Comment (tobiasschweizer) [inline docs/rst/04-salsah/development/build-process.rst]: @subotic Could you transfer these instructions to #873?

**Comment by subotic** [comment]
changes moved to https://github.com/dhlab-basel/Knora/pull/873

**Pull request closed without merging**